assets/js/event-providers/popup-maker.js

```diff
@@ -14,12 +14,149 @@
  * limitations under the License.
  */
 
+/**
+ * Internal dependencies
+ */
+import { PII_TYPE, normalizeValue, classifyPII } from './utils';
+
 ( ( jQuery, PUM ) => {
 	if ( ! jQuery || ! PUM ) {
 		return;
 	}
 
-	PUM.hooks.addAction( 'pum.integration.form.success', () => {
-		global._googlesitekit?.gtagEvent?.( 'submit_lead_form' );
+	PUM.hooks.addAction( 'pum.integration.form.success', ( event, args ) => {
+		const gtagUserDataEnabled = global._googlesitekit?.gtagUserData;
+
+		const userData =
+			gtagUserDataEnabled && shouldHandleProvider( args.formProvider )
+				? getUserData( event, args )
+				: undefined;
+
+		global._googlesitekit?.gtagEvent?.(
+			'submit_lead_form',
+			userData ? { user_data: userData } : undefined
+		);
 	} );
 } )( global.jQuery, global.PUM );
+
+const HANDLED_PROVIDERS = [ 'wpforms', 'contactform7', 'ninjaforms', 'mc4wp' ];
+
+/**
+ * Determines if the form provider is already handled in another script.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} provider The form provider (plugin) slug.
+ * @return {boolean} Whether this provider's submission should handled here or not.
+ */
+function shouldHandleProvider( provider ) {
+	return ! HANDLED_PROVIDERS.includes( provider );
+}
+
+/**
+ * Extracts and classifies user data from a form submission inside a Popup Maker popup.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} event A jQuery event object or a HTMLFormElement instance.
+ * @return {Object|undefined} A user_data object containing detected PII (address, email, phone_number), or undefined if no PII found.
+ */
+function getUserData( event ) {
```

Comment:
Same here. This function doesn't technically receive an event; instead, it receives a form element or a jQuery object. We should name it accordingly, and update its JSDoc reference.

```suggestion
/**
 * Extracts and classifies user data from a form submission inside a Popup Maker popup.
 *
 * @since n.e.x.t
 *
 * @param {Object} form A jQuery object or an HTMLFormElement instance.
 * @return {Object|undefined} A user_data object containing detected PII (address, email, phone_number), or undefined if no PII found.
 */
function getUserData( form ) {
```